app/src/main/java/com/gholem/moneylab/features/add/adapter/TransactionsAddAdapter.kt

```diff
@@ -0,0 +1,81 @@
+package com.gholem.moneylab.features.add.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.gholem.moneylab.R
+import com.gholem.moneylab.domain.model.AddFragmentDataModel
+
+class TransactionsAddAdapter : RecyclerView.Adapter<TransactionsAddAdapter.ViewHolder>() {
+
+    private val adapterData = mutableListOf<AddFragmentDataModel>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val layout = when (viewType) {
+            TYPE_CATEGORY -> R.layout.item_category
+            TYPE_TRANSACTION -> R.layout.item_transaction
+            TYPE_NEWTRANSACTION -> R.layout.item_new_transaction
+            else -> throw IllegalArgumentException("Invalid view type")
+        }
+
+        val view = LayoutInflater
```

Comment:
In the example of adapter with multiple ViewHolders I have sent you there was no `ViewBinding` used so this implementation is not consistent with the architecture of the app - here is the example of adapter with `ViewBinding` so please refactor it - https://www.simplifiedcoding.net/recyclerview-with-multiple-view-types/